proposals/0534-metro-package-exports-support.md

```diff
@@ -135,16 +135,15 @@ One edge case is a path conflict, for instance if the file tree for this package
 
 **Proposed**:
 
-- Metro will consider all exact `"exports"` subpaths when present.
-- In the case of a path conflict, when not using strict handling, Metro will prioritise the apparent file path over the explicit subpath if the file exists.
+- **Breaking**: Metro will consider all exact `"exports"` subpaths when present. In the case of a path conflict, Metro will prioritise the file specified by `"exports"`.
```

Comment:
Related: Do we specify anywhere what happens if there is a match in `exports` but it fails to resolve? (e.g. an already-published package has an `exports` field that happens to be broken, but currently works under Metro)

In non-strict mode, I think there's a case for falling back to filesystem resolution (with a warning), similar to the case where there isn't a match to begin with.

Reply:
Great point, I've amended this in 👍🏻